shapely/geometry/point.py

```diff
@@ -64,15 +64,19 @@ def __new__(self, *args):
             # Accept either (x, y) or [(x, y)]
             if not hasattr(coords, "__getitem__"):  # generators
                 coords = list(coords)
-
             coords = np.asarray(coords).squeeze()
         else:
             # 2 or 3 args
             coords = np.array(args).squeeze()
 
+        if coords.ndim > 1:
+            raise ValueError(
+                "Invalid values passed to Point constructor, "
+                "only size-1 arrays are accepted"
+            )
```

Comment:
It looks like this code isn't tested. We need a failing test to demonstrate that lines 72-76 are needed.

Reply:
It is tested in the original test, see discussion about it above:
https://github.com/shapely/shapely/pull/1720#discussion_r1071623475